.github/workflows/kind.yml

```diff
@@ -73,11 +73,13 @@ jobs:
 
       - name: Deploy assisted-migration
         run: |
-          make deploy-on-kind MIGRATION_PLANNER_API_IMAGE=$MIGRATION_PLANNER_API_IMAGE MIGRATION_PLANNER_AGENT_IMAGE=$MIGRATION_PLANNER_AGENT_IMAGE MIGRATION_PLANNER_API_IMAGE_PULL_POLICY=$MIGRATION_PLANNER_API_IMAGE_PULL_POLICY INSECURE_REGISTRY=$INSECURE_REGISTRY MIGRATION_PLANNER_NAMESPACE=default
+          make deploy-on-kind MIGRATION_PLANNER_API_IMAGE=$MIGRATION_PLANNER_API_IMAGE MIGRATION_PLANNER_AGENT_IMAGE=$MIGRATION_PLANNER_AGENT_IMAGE MIGRATION_PLANNER_API_IMAGE_PULL_POLICY=$MIGRATION_PLANNER_API_IMAGE_PULL_POLICY INSECURE_REGISTRY=$INSECURE_REGISTRY MIGRATION_PLANNER_NAMESPACE=default PERSISTENT_DISK_DEVICE=/dev/vda
           kubectl wait --for=condition=Ready pods --all --timeout=240s
           kubectl port-forward --address 0.0.0.0 service/migration-planner-agent 7443:7443 &
           kubectl port-forward --address 0.0.0.0 service/migration-planner 3443:3443 &
 
       - name: Run test
         run: |
+          mkdir /tmp/untarova
+          cp data/persistence-disk.qcow2 /tmp/untarova/persistence-disk.qcow2
```

Comment:
Can we create this file on the fly instead of storing in git please?

Reply:
done!